CHANGES.md

```diff
@@ -2,6 +2,15 @@
 
 ### 1.99 - 2022-11-01
 
+#### Major Announcements :loudspeaker:
+
+- Starting with Cesium 1.100, CesiumJS will be published alongside two smaller packages `@cesium/engine` and `@cesium/widgets` [#10824](https://github.com/CesiumGS/cesium/pull/10824):
+  - The source code will been paritioned into two folders: `packages/engine` and `packages/widgets`.
+  - These workspaces packages will follow semantic versioning.
+  - These workspaces packages will be published as ES modules with TypeScript definitions.
+  - The combined CesiumJS release will continue to be published, however, only the `Assets`, `ThirdParty` and `Widgets` (with CSS files only) are available in the root level `Source` folder.
+  - The ability to import modules and TypeScript definitions from individual files will been removed.
```

Comment:
> TypeScript definitions from individual files

Unless I misunderstand, I don't think this was previously possible? The types only existed at at top-level `Cesium.d.ts`.

Reply:
Also, I think it will help users to advise on how to import individual modules, (ie `import { Cartesian3 } from "cesium";` or `import { Cartesian3} from "Source/Cesium.js";`).

Reply:
Currently in `Cesium.d.ts`, once the `cesium` module is declared, the following lines of code also declare individual modules for all source files:

https://github.com/CesiumGS/cesium/blob/4aac68a5d6b3603e13c3885b526b94a78b79f429/gulpfile.js#L1468-L1481

This will not work anymore once we have the workspaces merged in.